java/src/main/java/com/codurance/training/tasks/ViewService.java

```diff
@@ -0,0 +1,8 @@
+package com.codurance.training.tasks;
+
+public interface ViewService {
+    void viewByDate();
+    void viewByDeadline();
+    void viewByProject();
+    void viewDueTasks();
```

Comment:
This violates interface seggregation principle... refactor it

Reply:
Addressed the comment..